catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergConfig.java

```diff
@@ -70,6 +71,21 @@ public class IcebergConfig extends Config {
           .booleanConf()
           .createWithDefault(true);
 
+  public static final ConfigEntry<String> ICEBERG_METRICS_STORAGE =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE)
+          .doc("The storage to store Iceberg metrics")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault("");
```

Comment:
What is the default value? From my understanding, I think you should have a default value.

Reply:
the default value is not store metrics from the view of user. We use EmptyMetricsStorage internally

Reply:
using `memory` for default